meeg_tools/utils/epochs.py

```diff
@@ -145,10 +145,11 @@ def create_metadata(epochs: Epochs):
                 ] = (epoch_ind + 1)
 
         metadata.loc[
-            metadata["id"].isin([44, 45, 46, 47, 144, 145, 146, 147]), "answer"
+            metadata["id"].isin([44, 45, 46, 47, 49, 144, 145, 146, 147, 149]), "answer"
```

Comment:
Increased list of incorrect answer with "slow response", this way those stimuli will be marked as incorrect that was followed by "slow response" coded as `49` or `149`